### DISCUSSION TITLE
Repeater with Associate logic

### DISCUSSION THREAD

**Discussion by MasterOfWeb** [filamentphp/filament#9798] [Help]
Hi

Am I missing something, or is it not possible to have a repeater for a simple hasMany relationship that is associates and dissociates the items? I get the repeater to work, but it will create new items or completely delete the associated items. This is not what I want.

I got this to work through a relationship-table, but this is a little inconvenient from UX side.

**Comment by classicalguss** [comment]
Same for me. I ended up using a RelationManager. But it seems to be a little over the top, just to add associate.
